oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/MoveVersionableNodeWithMemoryRepositoryTest.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.jackrabbit.oak.jcr;
```

Comment:
So, IIUC, you have created mutliple test classes so that the different mixtures are covered automatically. That was certainly helpful for testing, but it wouldn't scale if we did this everywhere.

So I would propose to simplify that again; and we also should wortk on the orthogonal issue of documenting and potentially fixing the CI pipelines.

Reply:
fixed in latest commit